<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Results>
    <UnitTestResult>
      <TestDate>2012-10-19T14:45:10</TestDate>
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test successful

Execution time: 0,10ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-10-19T14:58:26</TestDate>
      <Passed>1</Passed>
      <Errors>1</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-10-19T15:04:39</TestDate>
      <Passed>2</Passed>
      <Errors>1</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-10-19T15:08:29</TestDate>
      <Passed>2</Passed>
      <Errors>1</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
  </Results>
  <Tests>
    <TestRecord Name="UnitTests">
      <Results>
        <UnitTestResult>
          <TestDate>2012-10-19T14:45:10</TestDate>
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0,08ms</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Calc">
          <Results>
            <UnitTestResult>
              <TestDate>2012-10-19T14:45:10</TestDate>
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0,07ms</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="InvalidActionTest">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:45:10</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,01ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:45:59</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: &lt;System.NotImplementedException&gt;
  But was:  &lt;System.ArgumentException&gt; (action)
  at UnitTests.BarLogic.Calc (Char action, Int32 a, Int32 b) [0x00044] in /Users/kirill/Desktop/UnitTests/BarLogic.cs:28 
  at UnitTests.Calc+&lt;InvalidActionTest&gt;c__AnonStorey0.&lt;&gt;m__0 () [0x00000] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:15 
  at NUnit.Framework.Assert.Throws (IResolveConstraint expression, NUnit.Framework.TestDelegate code, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
</Message>
                  <StackTrace>  at NUnit.Framework.Assert.That (System.Object actual, IResolveConstraint expression, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
  at NUnit.Framework.Assert.Throws (IResolveConstraint expression, NUnit.Framework.TestDelegate code, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
  at NUnit.Framework.Assert.Throws (System.Type expectedExceptionType, NUnit.Framework.TestDelegate code) [0x00000] in &lt;filename unknown&gt;:0 
  at UnitTests.Calc.InvalidActionTest () [0x00013] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:14 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
    <TestRecord Name="BarLogicTests">
      <Results>
        <UnitTestResult>
          <TestDate>2012-10-19T14:58:26</TestDate>
          <Passed>1</Passed>
          <Errors>1</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-10-19T15:04:39</TestDate>
          <Passed>2</Passed>
          <Errors>1</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-10-19T15:08:29</TestDate>
          <Passed>2</Passed>
          <Errors>1</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-10-19T15:10:19</TestDate>
          <Passed>3</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0,71ms</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Calc">
          <Results>
            <UnitTestResult>
              <TestDate>2012-10-19T14:58:26</TestDate>
              <Passed>1</Passed>
              <Errors>1</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-10-19T15:04:39</TestDate>
              <Passed>2</Passed>
              <Errors>1</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-10-19T15:08:29</TestDate>
              <Passed>2</Passed>
              <Errors>1</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-10-19T15:10:19</TestDate>
              <Passed>3</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0,62ms</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="InvalidActionTest">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:58:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,01ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:04:39</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,01ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:08:29</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,02ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:10:19</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,02ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="PlusNotShow">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:58:26</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Moq.MockException : ICalculateService.Plus(5, 10) invocation failed with mock behavior Strict.
All invocations on the mock must have a corresponding setup.</Message>
                  <StackTrace>  at Moq.Interceptor.Intercept (ICallContext invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Proxy.CastleProxyFactory+Interceptor.Intercept (IInvocation invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.DynamicProxy.AbstractInvocation.Proceed () [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.Proxies.ICalculateServiceProxy.Plus (Int32 a, Int32 b) [0x00000] in &lt;filename unknown&gt;:0 
  at UnitTests.BarLogic.Calc (Char action, Int32 a, Int32 b, Boolean show) [0x0001e] in /Users/kirill/Desktop/UnitTests/BarLogic.cs:22 
  at BarLogicTests.Calc.PlusNotShow () [0x000d9] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:35 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:58:34</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Moq.MockException : ICalculateService.Plus(5, 10) invocation failed with mock behavior Strict.
All invocations on the mock must have a corresponding setup.</Message>
                  <StackTrace>  at Moq.Interceptor.Intercept (ICallContext invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Proxy.CastleProxyFactory+Interceptor.Intercept (IInvocation invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.DynamicProxy.AbstractInvocation.Proceed () [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.Proxies.ICalculateServiceProxy.Plus (Int32 a, Int32 b) [0x00000] in &lt;filename unknown&gt;:0 
  at UnitTests.BarLogic.Calc (Char action, Int32 a, Int32 b, Boolean show) [0x0001e] in /Users/kirill/Desktop/UnitTests/BarLogic.cs:22 
  at BarLogicTests.Calc.PlusNotShow () [0x000d9] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:35 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T14:59:09</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,29ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:00:10</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,29ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:04:39</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,06ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:08:29</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,02ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:10:19</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,01ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="MinusWithShow">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:04:39</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Moq.MockException : ICalculateService.Minus(12, 5) invocation failed with mock behavior Strict.
All invocations on the mock must have a corresponding setup.</Message>
                  <StackTrace>  at Moq.Interceptor.Intercept (ICallContext invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Proxy.CastleProxyFactory+Interceptor.Intercept (IInvocation invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.DynamicProxy.AbstractInvocation.Proceed () [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.Proxies.ICalculateServiceProxy.Minus (Int32 a, Int32 b) [0x00000] in &lt;filename unknown&gt;:0 
  at UnitTests.BarLogic.Calc (Char action, Int32 a, Int32 b, Boolean show) [0x00031] in /Users/kirill/Desktop/UnitTests/BarLogic.cs:25 
  at BarLogicTests.Calc.MinusWithShow () [0x00013] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:50 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:08:29</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Moq.MockException : 
Expected invocation on the mock at least once, but was never performed: m =&gt; m.OutputBufer = "7"
No setups configured.
No invocations performed.</Message>
                  <StackTrace>  at Moq.Mock.ThrowVerifyException (Moq.MethodCall expected, IEnumerable`1 setups, IEnumerable`1 actualCalls, System.Linq.Expressions.Expression expression, Times times, Int32 callCount) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Mock.VerifyCalls (Moq.Interceptor targetInterceptor, Moq.MethodCall expected, System.Linq.Expressions.Expression expression, Times times) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Mock.VerifySet[IOutputService] (Moq.Mock`1 mock, System.Action`1 setterExpression, Times times, System.String failMessage) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Mock`1[UnitTests.IOutputService].VerifySet (System.Action`1 setterExpression) [0x00000] in &lt;filename unknown&gt;:0 
  at BarLogicTests.Calc.MinusWithShow () [0x00081] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:49 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:09:10</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Moq.MockException : IOutputService.Show() invocation failed with mock behavior Strict.
All invocations on the mock must have a corresponding setup.</Message>
                  <StackTrace>  at Moq.Interceptor.Intercept (ICallContext invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Moq.Proxy.CastleProxyFactory+Interceptor.Intercept (IInvocation invocation) [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.DynamicProxy.AbstractInvocation.Proceed () [0x00000] in &lt;filename unknown&gt;:0 
  at Castle.Proxies.IOutputServiceProxy.Show () [0x00000] in &lt;filename unknown&gt;:0 
  at UnitTests.BarLogic.Calc (Char action, Int32 a, Int32 b, Boolean show) [0x00068] in /Users/kirill/Desktop/UnitTests/BarLogic.cs:34 
  at BarLogicTests.Calc.MinusWithShow () [0x000d9] in /Users/kirill/Desktop/UnitTests/BarLogic/Calc.cs:53 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:10:14</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,44ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-10-19T15:10:19</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0,42ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>